config/locales/en.yml

```
@@ -100,6 +100,7 @@
       can_order_devices: Yes, give them access to the TechSource website
       can_order_devices_hint: This is where they’ll place orders.
       cannot_order_devices: 'No'
+    new_school_user: New school user
```

```suggestion
    new_school_user: Invite a new user
```
This copy is consistent with the responsible body and school versions of this apge.

---

done